### DISCUSSION TITLE
[Bug]: Frigate 0.14.0 Review Motion Window

### DISCUSSION THREAD

**Discussion by rourke750** [blakeblackshear/frigate#11525] [General Support] [answered]
### Describe the problem you are having

When clicking on the motion tab of the review page I get a spinning wheel and nothing shows up.

### Version

0.14.0 beta

### Frigate config file

```yaml
Same as other
```

### Relevant log output

```shell
pandas/core/dtypes/astype.py:133 RunetimeWarning: overflow encountered in cast
...
/opt/frigate/frigate/api/review.py:467: RuntimeWarning: invalid value encountered in scalar subtract
/ (part["motion"].max() - part["motion"].min())
```

### FFprobe output from your camera

```shell
na
```

### Operating system

Debian

### Install method

Docker Compose

### Coral version

USB

### Network connection

Wired

### Camera make and model

na

**Comment by hawkeye217** [comment, marked as answer]
If you just started using the new UI, there is no motion data to view yet. This could probably be handled better, but give Frigate a few hours of runtime and come back to it then!

**Reply by rourke750** [reply]
If I record motion shouldn't it still show up?  It does appear to be working now but I did have Alerts and motion wasn't there.  Seems to be working now though

**Reply by hawkeye217** [reply]
Great, glad to hear. This is expected behavior because of how we deal with aggregating motion in the backend.

**Comment by hawkeye217** [comment]
Fixed by https://github.com/blakeblackshear/frigate/pull/11593
